<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="opcion1">
      <property name="text">
       <string>Opcion 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="opcion2">
      <property name="text">
       <string>Opcion 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="opcion3">
      <property name="text">
       <string>Opcion 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="seleccion">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
